data/json/recipes/recipe_food.json

```
@@ -2062,7 +2062,7 @@
     "difficulty": 4,
     "time": "20 m",
     "batch_time_factors": [ 50, 4 ],
-    "autolearn": true,
+    "book_learn": [ [ "baking_book", 3 ] ],
```

```suggestion
    "book_learn": [ [ "cookbook_daintydishes", 6 ], [ "baking_book", 3 ] ],
```
Dunno what you think but I like the idea of daintydishes having mostly the same recipes as baking_book but at a way higher skill level due to all the silly flourishes